File: 1-easy/problems/2529-maximum-count-of-positive-integer-and-negative-integer/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def maximumCount(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        return max(bisect.bisect_left(nums, 0)-0, len(nums)-bisect.bisect_left(nums, 1))

File: 1-easy/problems/2529-maximum-count-of-positive-integer-and-negative-integer/solution.ts
function maximumCount(nums: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :rtype: int
  """
  return max(bisect.bisect_left(nums, 0)-0, (nums).length-bisect.bisect_left(nums, 1));
}

export { maximumCount };
